tiny_skia/src/text.rs

```diff
@@ -405,7 +408,10 @@ impl Cache {
         }
 
         if let hash_map::Entry::Vacant(entry) = self.entries.entry(hash) {
-            let metrics = cosmic_text::Metrics::new(key.size, key.size * 1.2);
+            let metrics = cosmic_text::Metrics::new(
+                key.size,
+                (key.size * 1.2).max(f32::MIN_POSITIVE),
```

Comment:
You found a bug! This should be the `key.line_height`!

Reply:
Oh that's great! I was actually confused why `key.size` was used there too. Should I change it to line height?

Reply:
Yes!